src/main/java/uk/gov/hmcts/reform/sendletter/config/ReportsServiceConfig.java

```diff
@@ -2,20 +2,51 @@
 
 import org.springframework.boot.context.properties.ConfigurationProperties;
 
+import java.util.List;
 import java.util.Map;
 import java.util.Optional;
 
+import static java.util.stream.Collectors.toMap;
+
 @ConfigurationProperties(prefix = "reports")
 public class ReportsServiceConfig {
 
     private Map<String, String> serviceConfig;
 
+    public static class Mapping {
+
+        private String service;
+        private String displayName;
+
+        public Mapping() {
+            // Spring needs it.
+        }
+
+        public String getService() {
+            return service;
+        }
+
+        public void setService(String service) {
+            this.service = service;
+        }
+
+        public String getDisplayName() {
+            return displayName;
+        }
+
+        public void setDisplayName(String displayName) {
+            this.displayName = displayName;
+        }
+    }
+
     public Map<String, String> getServiceConfig() {
         return serviceConfig;
     }
 
-    public void setServiceConfig(Map<String, String> serviceConfig) {
-        this.serviceConfig = serviceConfig;
+    public void setServiceConfig(List<Mapping> mappings) {
+        this.serviceConfig = mappings
```

Comment:
perhaps we can just keep the list of object with clear names instead of converting to a `Map<String, String>`?